storage/google/cloud/storage/blob.py

```diff
@@ -1109,7 +1145,7 @@ def create_resumable_upload_session(
             # to the `ResumableUpload` constructor. The chunk size only
             # matters when **sending** bytes to an upload.
             upload, _ = self._initiate_resumable_upload(
-                client, dummy_stream, content_type, size, None,
+                client, dummy_stream, content_type, size, None, None,
```

Comment:
For clarity, can we use keyword arguments for the two `None` values here?

Reply:
They are actually positional args.  Let's just fix this in another PR for clarity purposes.